time/timeswitch/timeswitch.js

```diff
@@ -14,6 +15,7 @@ module.exports = function(RED) {
         this.duskoff = n.duskoff;
         this.dawnoff = n.dawnoff;
         this.mytopic = n.mytopic;
+        this.timezone = n.timezone;
```

Comment:
Should this default correctly so that old flows work ? I thin line 17 will barf if it's undefined - should force default to UTC if not set

Reply:
Good catch! Yes, UTC is what the previous version of the node used, so I just checked in a version that makes that the default if undefined.

Reply:
great.